ding/framework/middleware/functional/evaluator.py

```diff
@@ -208,6 +281,12 @@ def _evaluate(ctx: "OnlineRLContext"):
         ctx.last_eval_iter = ctx.train_iter
         ctx.eval_value = eval_reward
         ctx.eval_output = {'output': output, 'reward': episode_reward}
+        episode_info = eval_monitor.get_episode_info()
+        if episode_info is not None:
+            ctx.eval_output['episode_info'] = episode_info
+        if render:
+            ctx.eval_output['replay_video'] = eval_monitor.get_episode_video()
+            ctx.eval_output['output'] = eval_monitor.get_episode_output()
```

Comment:
the 'output' cover the output in L283, why not 'render_output'

Reply:
modify implementation, use different if-else branch to ensure compatibility